samples/matter/lock/src/app_task.h

```diff
@@ -17,6 +17,10 @@ struct Identify;
 enum class SwitchButtonAction : uint8_t { Pressed, Released };
 #endif
 
+#ifdef CONFIG_NCS_SAMPLE_MATTER_TEST_EVENT_TRIGGERS_REGISTER_DEFAULTS
```

Comment:
ditto, application code should not depend on `CONFIG_NCS_SAMPLE_MATTER_TEST_EVENT_TRIGGERS_REGISTER_DEFAULTS`